clients/js/solana.ts

```diff
@@ -67,7 +77,7 @@ export async function execute_solana(
           break
         case "Transfer":
           console.log("Completing transfer")
-          if (payload.tokenChain === CHAIN_ID_SOLANA) {
+          if (payload.tokenChain === CHAINS[chain]) {
```

Comment:
Should we check if this is solana or pyth?

Reply:
It is in CHAINS[]